.github/workflows/build.yml

```diff
@@ -49,9 +58,26 @@ jobs:
           free -m
           docker system df
           docker system df -v
-      - name: Build EVE KVM
+
+  eve:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        arch: [arm64, amd64]
+        hv: [xen, eve]
```

Comment:
Should it be kvm instead of eve?

Reply:
Shoot, yes, nice catch. Way too much copy-paste. I am fixing.

Reply:
Fixed